run-tests.sh

```diff
@@ -27,6 +27,6 @@ checkError
 
 checkError
 
-./gradlew --no-daemon -PmavenLocal --continue clean check smokeTests --scan
+GITHUB_CI=true ./gradlew --no-daemon -PmavenLocal --continue clean check smokeTests --scan
```

Comment:
What does this do in front of a shell script?

Reply:
it sets an environmental variable.  I guess we could add it in yaml file instead, incase someone runs 'run-tests.sh' locally.